runtime/jvmti/jvmtiHelpers.c

```diff
@@ -875,6 +875,9 @@ getVirtualThreadState(J9VMThread *currentThread, jthread thread)
 				Assert_JVMTI_unreachable();
 				rc = JVMTI_ERROR_INTERNAL;
 			}
+			if (0 != J9OBJECT_U32_LOAD(currentThread, vThreadObject, vm->isSuspendedByJVMTIOffset)) {
```

Comment:
Is there a case missing here (after the vthread mount clears this flag and sets the publicFlag then blocks).

Reply:
It should fall under the vthread mounted case, which is covered by the `if (NULL != targetThread)` block.

Reply:
What is the sequence for mounting/unmounting (regarding the start/end/enter/yield natives).